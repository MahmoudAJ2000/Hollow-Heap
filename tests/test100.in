100 300 100 49
14 35 519
88 87 514
45 18 106
29 95 55
10 52 466
22 8 955
59 51 797
30 11 422
65 1 190
43 35 564
22 43 763
27 91 396
19 45 826
2 17 496
77 16 699
59 82 134
43 68 563
40 66 846
18 98 682
50 51 208
96 24 195
57 3 937
34 36 784
18 74 966
96 81 155
12 17 779
21 39 887
63 70 835
70 56 333
86 94 223
68 82 13
7 50 125
26 69 421
43 51 559
77 50 833
4 67 941
54 57 332
58 77 113
64 29 867
72 18 363
42 90 326
29 19 442
54 21 619
17 2 226
72 33 818
93 89 983
1 93 870
64 42 603
73 26 288
25 53 981
14 51 114
83 26 662
34 6 639
59 10 220
54 24 612
43 52 709
27 69 522
52 86 294
9 33 460
86 50 677
99 90 716
58 32 103
92 32 994
81 19 445
57 15 441
62 48 734
87 49 866
79 83 625
95 66 448
71 0 568
42 56 835
56 62 742
79 63 786
61 92 591
17 95 327
8 92 402
89 16 868
72 41 929
50 44 574
47 77 909
30 45 494
80 84 904
74 39 853
21 52 668
54 59 783
75 60 887
83 8 94
16 14 629
30 2 493
4 66 815
92 26 402
6 89 165
65 22 105
83 48 695
37 44 457
57 96 486
40 39 783
86 60 506
39 42 975
26 88 370
56 89 412
8 94 98
89 83 55
24 75 324
91 47 213
6 11 79
58 98 747
98 43 751
31 10 484
90 36 531
64 55 750
41 74 335
84 5 469
79 94 747
17 60 473
56 61 29
63 88 586
48 71 218
22 65 972
26 98 245
70 80 565
55 62 429
51 41 193
2 73 438
87 60 233
76 38 384
78 76 736
27 3 699
85 69 168
16 84 434
99 24 484
34 90 728
5 52 206
16 34 312
60 2 67
39 19 377
73 95 288
82 96 828
92 63 515
29 3 296
31 56 559
68 88 889
94 50 163
90 19 618
39 4 597
91 24 900
60 88 768
56 47 498
25 70 406
75 26 199
92 48 6
70 38 207
28 72 413
38 70 809
38 10 669
27 16 672
41 44 8
65 63 393
49 22 366
95 61 416
52 57 640
41 23 759
98 30 310
33 43 124
66 86 161
75 92 941
82 86 484
38 99 673
75 42 386
31 48 667
97 87 274
99 57 696
67 98 724
27 56 451
90 46 737
45 12 887
14 24 968
37 41 999
64 9 808
78 61 898
22 57 981
26 28 407
58 85 726
86 28 635
44 62 736
44 16 13
92 27 863
98 71 966
51 52 685
29 14 214
32 12 844
8 77 4
53 99 717
65 39 368
47 39 35
55 75 494
91 62 397
16 97 580
18 14 709
45 75 814
9 32 786
88 86 846
37 32 463
59 22 586
29 13 576
97 48 284
30 43 661
2 15 850
28 23 870
69 33 434
29 46 744
22 36 252
48 33 406
94 33 66
21 20 465
55 80 703
21 95 733
52 41 167
92 84 772
4 88 272
19 17 305
75 32 227
84 23 215
81 6 511
10 80 212
12 9 713
56 29 423
60 5 400
80 27 262
54 12 417
3 49 438
99 6 231
86 41 265
31 62 367
24 2 41
67 86 297
30 15 561
15 55 759
22 4 423
21 86 223
56 19 639
86 34 304
17 24 373
74 38 83
9 24 700
47 78 184
26 47 78
87 80 412
41 72 824
86 83 909
83 55 976
59 23 89
98 97 356
95 25 523
2 16 153
18 10 302
47 18 352
99 73 95
89 59 371
85 76 42
84 18 285
47 13 740
82 58 803
49 0 928
34 66 753
78 74 266
67 87 754
40 21 818
2 95 587
17 66 938
21 92 490
62 26 222
49 27 798
59 74 968
62 30 150
39 25 689
62 97 521
74 50 546
17 13 658
77 47 501
5 66 478
43 12 696
28 71 377
19 36 380
4 94 820
3 18 250
91 31 230
27 97 829
21 98 878
59 15 828
84 75 33
55 26 733
43 10 332
22 34 156
41 15 521
34 77 483
90 38 173
65 93 780
26 20 160
97 68 595
55
54
47
50
43
75
12
71
89
7
28
56
27
57
11
15
69
85
64
82
94
33
37
68
97
22
13
78
88
46
30
63
23
4
62
0
76
38
86
60
26
1
72
52
80
32
25
93
70
51
18
24
81
53
67
5
90
48
35
42
45
92
61
10
9
36
96
3
17
91
95
44
77
79
84
87
66
29
40
41
58
74
65
59
14
20
34
19
73
8
83
2
39
31
99
16
49
21
6
98
