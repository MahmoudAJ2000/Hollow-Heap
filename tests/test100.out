Distance from 49 to 55 is 2357
No path from 49 to 54 was found
Distance from 49 to 47 is 1407
Distance from 49 to 50 is 1582
Distance from 49 to 43 is 1129
Distance from 49 to 75 is 1937
Distance from 49 to 12 is 1825
Distance from 49 to 71 is 1947
Distance from 49 to 89 is 1326
No path from 49 to 7 was found
Distance from 49 to 28 is 2071
Distance from 49 to 56 is 1249
Distance from 49 to 27 is 798
Distance from 49 to 57 is 1347
Distance from 49 to 11 is 1240
Distance from 49 to 15 is 1788
Distance from 49 to 69 is 1320
Distance from 49 to 85 is 3234
No path from 49 to 64 was found
Distance from 49 to 82 is 1705
Distance from 49 to 94 is 1419
Distance from 49 to 33 is 1485
No path from 49 to 37 was found
Distance from 49 to 68 is 1692
Distance from 49 to 97 is 1627
Distance from 49 to 22 is 366
Distance from 49 to 13 is 2147
Distance from 49 to 78 is 1591
Distance from 49 to 88 is 1061
Distance from 49 to 46 is 1987
Distance from 49 to 30 is 1741
Distance from 49 to 63 is 1731
Distance from 49 to 23 is 1786
Distance from 49 to 4 is 789
Distance from 49 to 62 is 1591
Distance from 49 to 0 is 928
Distance from 49 to 76 is 2327
Distance from 49 to 38 is 1423
Distance from 49 to 86 is 1436
Distance from 49 to 60 is 1808
Distance from 49 to 26 is 1813
Distance from 49 to 1 is 1528
Distance from 49 to 72 is 2484
Distance from 49 to 52 is 1838
Distance from 49 to 80 is 1673
Distance from 49 to 32 is 2164
Distance from 49 to 25 is 2131
Distance from 49 to 93 is 2118
Distance from 49 to 70 is 2232
Distance from 49 to 51 is 1688
Distance from 49 to 18 is 1747
Distance from 49 to 24 is 2028
Distance from 49 to 81 is 1988
Distance from 49 to 53 is 3112
Distance from 49 to 67 is 1730
Distance from 49 to 5 is 2208
Distance from 49 to 90 is 1250
Distance from 49 to 48 is 1729
Distance from 49 to 35 is 1693
Distance from 49 to 42 is 2323
Distance from 49 to 45 is 2235
Distance from 49 to 92 is 1723
Distance from 49 to 61 is 1278
Distance from 49 to 10 is 1461
Distance from 49 to 9 is 2538
Distance from 49 to 36 is 618
Distance from 49 to 96 is 1833
Distance from 49 to 3 is 1497
Distance from 49 to 17 is 2124
Distance from 49 to 91 is 1194
Distance from 49 to 95 is 1727
Distance from 49 to 44 is 1709
Distance from 49 to 77 is 1005
No path from 49 to 79 was found
Distance from 49 to 84 is 1904
Distance from 49 to 87 is 1575
Distance from 49 to 66 is 1275
Distance from 49 to 29 is 1672
No path from 49 to 40 was found
Distance from 49 to 41 is 1701
Distance from 49 to 58 is 2508
Distance from 49 to 74 is 1857
Distance from 49 to 65 is 1338
Distance from 49 to 59 is 1697
Distance from 49 to 14 is 1886
Distance from 49 to 20 is 1973
Distance from 49 to 34 is 522
Distance from 49 to 19 is 1819
Distance from 49 to 73 is 2191
Distance from 49 to 8 is 1321
Distance from 49 to 83 is 1381
Distance from 49 to 2 is 1875
Distance from 49 to 39 is 1442
Distance from 49 to 31 is 1424
Distance from 49 to 99 is 2096
Distance from 49 to 16 is 1470
Distance from 49 to 49 is 0
No path from 49 to 21 was found
Distance from 49 to 6 is 1161
Distance from 49 to 98 is 2058
